spec/factories/removal_order_problems.rb

```diff
@@ -1,6 +1,7 @@
 FactoryBot.define do
   factory :removal_order_problem do
     user { nil }
```

Comment:
Pra arrumar a factory você poderia fazer algo assim

```suggestion
    ownable { create(:user) }
```